ara/api/models.py

```diff
@@ -240,6 +240,27 @@ def __str__(self):
         return "<Host %s:%s>" % (self.id, self.name)
 
 
+class DistinctHost(models.Model):
+    """
+    Distinct list of hosts with latest `Host` object id related.
+
+    We can not inherit from Base because we want to use `name` as primary key.
+    """
+
+    class Meta:
+        db_table = "distinct_hosts"
+
+    name = models.CharField(max_length=255, primary_key=True)
+    latest_host = models.ForeignKey(Host, on_delete=models.CASCADE)
```

Comment:
Just a thought: maybe we should name this ``latest`` instead of ``latest_host`` ?

An example usage in a template:
![Screenshot from 2021-06-18 20-09-07](https://user-images.githubusercontent.com/1291204/122625031-11982e80-d071-11eb-94b6-40e98802b286.png)

So, for example, ``host.latest_host.id`` would become ``host.latest.id`` and so on.

Reply:
sure, we can just rename it. Makes definitely sense.

Reply:
changed in 60f427f